### PR TITLE
{aks} aks-preview vendor 2021-08-01 SDK

### PR DESCRIPTION
**Pull request by chengliangli0918**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/tests/latest/data/httpproxyconfig.json [hunk 1]
```diff
@@ -4,6 +4,5 @@
     "noProxy": [
         "localhost",
         "127.0.0.1"
-    ],
-    "trustedCa": "G9yR2xrQ29NRjNURHg4cm1wOURCaUIvCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0="
```
Comment (chengliangli0918) [inline src/aks-preview/azext_aks_preview/tests/latest/data/httpproxyconfig.json:8]: I confirmed with Node Provisioning team, currently HttpProxy config with trustedCA doesn't work. So generate this recording test without trustedCA.

**Comment by yonzhan** [comment]
aks

**Review by FumingZhang — COMMENTED** [review]
We have 18 test cases marked @live_only and it is not necessary to add recording files for them. Could you please remove them?

**Comment by chengliangli0918** [comment]
@FumingZhang, thanks for mentioning it. I restored those 18 recording tests marked with live_only.

> We have 18 test cases marked @live_only and it is not necessary to add recording files for them. Could you please remove them?

**Review by FumingZhang — APPROVED** [review]

**Pull request merged**